src/v0/destinations/braze/util.js

```diff
@@ -655,6 +655,46 @@ function getPurchaseObjs(message, config) {
   return purchaseObjs;
 }
 
+const collectStatsForAliasFailure = (brazeResponse, destinationId) => {
+  /**
+   * Braze Response for Alias failure
+   * {
+   * "aliases_processed": 0,
+   * "message": "success",
+   * "errors": [
+   *     {
+   *         "type": "'external_id' is required",
+   *         "input_array": "user_identifiers",
+   *         "index": 0
+   *     }
+   *   ]
+   * }
+   */
+
+  /**
+   * Braze Response for Alias success
+   * {
+   *   "aliases_processed": 1,
+   *   "message": "success"
+   *   }
+   */
+
+  const { aliases_processed: aliasesProcessed, errors } = brazeResponse;
+  if (aliasesProcessed === 0) {
+    stats.increment('braze_alias_failure_count', { destination_id: destinationId });
+    const errorType = [];
+    if (Array.isArray(errors)) {
+      errors.forEach((error) => {
+        errorType.push(error.type);
+      });
+    }
+    stats.increment('braze_alias_failure_error_type', {
+      destination_id: destinationId,
+      alias_error_type: errorType.join(','),
```

Comment:
this might increase the cardinality so push individual error types as separate metrics

Reply:
avoid this metric and use logs instead

Reply:
addressed

Reply:
<img width="1132" alt="Screenshot 2024-06-10 at 2 02 39 PM" src="https://github.com/rudderlabs/rudder-transformer/assets/25389167/4cbfe072-7126-4b15-b0ee-b343d99affcc">